src/System.Private.CoreLib/shared/System/Delegate.cs

```diff
@@ -0,0 +1,107 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.Serialization;
+
+namespace System
+{
+    public abstract partial class Delegate : ICloneable, ISerializable
+    {
+		internal Delegate()
+		{
+		}
+
+		public virtual object Clone() => MemberwiseClone();
+
+		public static Delegate Combine(Delegate a, Delegate b)
+		{
+			if (a is null)
+				return b;
+
+			return a.CombineImpl(b);
+		}
+
+		public static Delegate Combine(params Delegate[] delegates)
+		{
+			if (delegates == null || delegates.Length == 0)
```

Comment:
Nit: it'd be good to be consistent, at least within the same file or least overloads of the same method, about how we do null checks.  You changed the above to be `a is null`... can you change this one to be `delegates is null`, too?

Reply:
done